modules/ROOT/pages/streaming-about.adoc

```diff
@@ -271,6 +271,18 @@ increments at 100 objects per increment and allows a maximum buffer size of
  </sfdc:query>
 ----
 
+[[non_repeatable_iterable]]
+=== Non-Repeatable Iterable
+
+This non-repeatable strategy disables repeatable streams, which enables you to write a stream of output objects only once. Use this strategy when your use case does not require the extra memory or performance overhead that comes with repeatable streaming. Because the stream is not saved to memory, the non-repeatable strategy performs better than the other strategies.
```

Comment:
This entire paragraph is wrong, but so is the non-repeatable-stream one. I would collapse both sections into a unique one about disable repeatable streaming that reads like this:

```
Repeatable streaming can be disabled through the non-repeatable-stream or non-repeatable-iterable strategies, depending on the type of stream.

This option should only be used when you are certain that there's no need to consume the stream several times and you need a very tight optimization for performance and resource consumption. 

When making that decision, you should consider the following: 

* Although disable repeatable streaming will improve performance, the significance of that gain will be proportional to the size of the streams themselves. Unless you're dealing with large streams, chances are that there are others things you can improve in your app that will provide a better optimization.
* Disabling repeatable streaming means that the stream can only be read once. When trying to determine if that's the case, keep in mind that your flow may contain components that require to consume the full stream even if you don't explicitly specify that. Examples include the <ee:cache> component, <ee:transform> (depending on the actual transformation being used), <foreach> (when trying to iterate through a Json array), or any other component using an expression which accesses the stream. 
```

Reply:
added and copyedited per style guidelines